# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e-utils/xlib.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- e_utils/src/bin/e17setroot/e17setroot.c_orig	2006-01-23 22:46:52.000000000 +0100
+++ e_utils/src/bin/e17setroot/e17setroot.c	2006-01-23 22:47:13.000000000 +0100
@@ -5,6 +5,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <limits.h>
+#include <X11/Xlib.h>
 #include <Imlib2.h>
 #include <E_Lib.h>
 #include <Engrave.h>
